src/containers/ConversionForm/custom-request.ts

```diff
@@ -1,19 +1,21 @@
-import {
-    UploadRequestOption,
-} from "rc-upload/lib/interface";
+import { UploadRequestOption } from "rc-upload/lib/interface";
 import { ViewerError } from "../../state/viewer/types";
 import { ErrorLevel, FrontEndError } from "@aics/simularium-viewer";
 import { ReceiveFileToConvertAction } from "../../state/trajectory/types";
 
 export default async (
     fileToConvert: any,
-    receiveFileToConvert: (file: string) => ReceiveFileToConvertAction,
+    receiveFileToConvert: (
+        fileContents: string,
+        fileName: string
+    ) => ReceiveFileToConvertAction,
     setError: (error: ViewerError) => void,
     rcRequest?: UploadRequestOption
 ) => {
     try {
         const unpackedFileText = await fileToConvert.originFileObj.text();
-        receiveFileToConvert(unpackedFileText);
+        const fileName = fileToConvert.name.replace(/\.[^/.]+$/, "");
```

Comment:
why is this needed? can you add a comment about what this regex selects for. You can also always use a named variable for your regex, like `const whiteSpace = RegEX; string.replace(whiteSpace, "")`